issues with search guard deployment

i tried to test the chart using search guard, all the elasticsearch pods fails because the sg plugins cannot be found:

```
org.elasticsearch.bootstrap.StartupException: org.elasticsearch.bootstrap.BootstrapException: java.nio.file.NoSuchFileException: /usr/share/elasticsearch/plugins/search-guard-6/plugin-descriptor.properties
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:134) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:121) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:69) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:134) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.cli.Command.main(Command.java:90) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:92) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:85) ~[elasticsearch-6.0.1.jar:6.0.1]
Caused by: org.elasticsearch.bootstrap.BootstrapException: java.nio.file.NoSuchFileException: /usr/share/elasticsearch/plugins/search-guard-6/plugin-descriptor.properties
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:168) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:322) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:130) ~[elasticsearch-6.0.1.jar:6.0.1]
	... 6 more
Caused by: java.nio.file.NoSuchFileException: /usr/share/elasticsearch/plugins/search-guard-6/plugin-descriptor.properties
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86) ~[?:1.8.0_151]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102) ~[?:1.8.0_151]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107) ~[?:1.8.0_151]
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214) ~[?:1.8.0_151]
	at java.nio.file.Files.newByteChannel(Files.java:361) ~[?:1.8.0_151]
	at java.nio.file.Files.newByteChannel(Files.java:407) ~[?:1.8.0_151]
	at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:384) ~[?:1.8.0_151]
	at java.nio.file.Files.newInputStream(Files.java:152) ~[?:1.8.0_151]
	at org.elasticsearch.plugins.PluginInfo.readFromProperties(PluginInfo.java:123) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Spawner.spawnNativePluginControllers(Spawner.java:80) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:166) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:322) ~[elasticsearch-6.0.1.jar:6.0.1]
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:130) ~[elasticsearch-6.0.1.jar:6.0.1]
	... 6 more
```

--- a/values.yaml
+++ b/values.yaml
@@ -1,7 +1,7 @@
 common:
   image:
     repository: blacktop/elasticsearch
-    tag: "6.0"
+    tag: "6.1.2"
     pullPolicy: IfNotPresent
 
   # Defines the service type for all outward-facing (non-discovery) services.
@@ -128,8 +128,9 @@ tls:
   imagePullPolicy: "IfNotPresent"
 
 searchguard:
+  # If you enable searchguard, you must enable TLS as well, and pay close attention to the instructions after install
   enable: false
-  plugin: com.floragunn:search-guard-6:6.1.0-20.0
+  plugin: com.floragunn:search-guard-6:6.1.2-20.0
   enterprise_modules: false
   kibana_user: kibana
   admins:

